src/icons/PermMedia.tsx

```diff
@@ -0,0 +1,13 @@
+import type React from "react";
```

Comment:
It is not obvious for me what `PermMedia` is. Could this have a name that is more descriptive?

Reply:
Lolz, I know what you mean - https://mui.com/material-ui/material-icons/?query=perm&selected=PermMedia